index.html

```diff
@@ -341,12 +339,12 @@ <h2 class="javaS">CSS</h2>
             <img
               class="contactme-img"
               src="Microverse-Student-Project-1/Contact.png"
-              alt=""
+              alt="background-img"
             />
             <img
               class="contact-shapes"
               src="Microverse-Student-Project-1/Contact2.png"
-              alt=""
+              alt="background-img"
```

Comment:
add some alt information to the image to increase the accessibility for people who don't want to load the images